custom_components/octopus_energy/heat_pump/sensor_lifetime_scop.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime
+import logging
+from typing import List
+
+from homeassistant.const import (
+    STATE_UNAVAILABLE,
+    STATE_UNKNOWN,
+    UnitOfPower
+)
+from homeassistant.core import HomeAssistant, callback
+
+from homeassistant.util.dt import (now)
+from homeassistant.helpers.update_coordinator import (
+  CoordinatorEntity
+)
+from homeassistant.components.sensor import (
+  RestoreSensor,
+  SensorDeviceClass,
+  SensorStateClass,
+)
+
+from .base import (BaseOctopusEnergyHeatPumpSensor)
+from ..utils.attributes import dict_to_typed_dict
+from ..api_client.heat_pump import HeatPump
+from ..coordinators.heat_pump_configuration_and_status import HeatPumpCoordinatorResult
+
+_LOGGER = logging.getLogger(__name__)
+
+class OctopusEnergyHeatPumpSensorLifetimeSCoP(CoordinatorEntity, BaseOctopusEnergyHeatPumpSensor, RestoreSensor):
+  """Sensor for displaying the lifetime SCoP of a heat pump."""
+
+  def __init__(self, hass: HomeAssistant, coordinator, heat_pump_id: str, heat_pump: HeatPump):
+    """Init sensor."""
+    # Pass coordinator to base class
+    CoordinatorEntity.__init__(self, coordinator)
+    BaseOctopusEnergyHeatPumpSensor.__init__(self, hass, heat_pump_id, heat_pump)
+
+    self._state = None
+    self._last_updated = None
+
+  @property
+  def unique_id(self):
+    """The id of the sensor."""
+    return f"octopus_energy_heat_pump_{self._heat_pump_id}_lifetime_scop"
+
+  @property
+  def name(self):
+    """Name of the sensor."""
+    return f"Lifetime SCoP ({self._heat_pump_id})"
```

Comment:
issue: Should follow convention of naming in other sensors. Should be

`Lifetime SCop Heat Pump ({self._heat_pump_id})`